### PR TITLE
Daylight | Add output connectors to the RGB sticks

### PR DESCRIPTION
**Pull request by bartlammers**
Suggestion by DarkNeutrino that has added value.

Provided pick and place files for all varieties as well. 
Created individual README.md files with BOM and changelog.

### CODE REVIEW

**Comment by bartlammers** [comment]
@timmit99

File: Daylight/Daylight_on_a_matchstick/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Daylight on a matchstick
+
+12 White leds to accurately illuminate your V0 or other ant-sized printer.
+
+## BOM
+| Quantity | Value                        | Package |
+|----------|------------------------------|---------|
+|        4 | 100 Ohms Resistor            | 1210    |
+|       12 | White LED (4000K, 90+ CRI)   | 2835    |
+|        2 | JST-XH angled connector      | 2pin    |
```
Comment (timmit99) [inline Daylight/Daylight_on_a_matchstick/README.md:10]: Maybe add 2.5/2.54 pitch to the connector callout

File: Daylight/Daylight_on_a_stick/README.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Daylight on a stick
+
+18 White leds to accurately illuminate your Voron.
+
+## BOM
+| Quantity | Value                        | Package |
+|----------|------------------------------|---------|
+|        6 | 100 Ohms Resistor            | 1210    |
+|       18 | White LED (4000K, 90+ CRI)   | 2835    |
+|        2 | JST-XH angled connector      | 2pin    |
```
Comment (timmit99) [inline Daylight/Daylight_on_a_stick/README.md:10]: Maybe add 2.5/2.54 pitch to the connector callout

File: Daylight/Disco_on_a_stick/README.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# Disco on a stick
+
+18 Addressable RGB leds to for a party in your Voron.
+
+## BOM
+| Quantity | Value                         | Package |
+|----------|-------------------------------|---------|
+|       15 | 100nF Capacitor               | 0805    |
+|       18 | Addressable RGB LED (WS2812B) | 5050    |
+|        2 | JST-XH angled connector       | 3pin    |
```
Comment (timmit99) [inline Daylight/Disco_on_a_stick/README.md:10]: Maybe add 2.5/2.54 pitch to the connector callout

File: Daylight/Rainbow_on_a_matchstick/README.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
+# Rainbow on a matchstick
+
+10 Addressable RGB leds to for a party in your V0 or other ant-sized printer.
+
+## BOM
+| Quantity | Value                         | Package |
+|----------|-------------------------------|---------|
+|        7 | 100nF Capacitor               | 0805    |
+|       10 | Addressable RGB LED (WS2812B) | 5050    |
+|        2 | JST-XH angled connector       | 3pin    |
```
Comment (timmit99) [inline Daylight/Rainbow_on_a_matchstick/README.md:10]: Maybe add 2.5/2.54 pitch to the connector callout

**Review by timmit99 — COMMENTED** [review]
Only comment would be to add a pitch callout to the connectors. Otherwise, looks good!

**Comment by bartlammers** [comment]
@timmit99 added (2.54) to the BOM lines

**Review by timmit99 — APPROVED** [review]

**Comment by timmit99** [comment]
@raymondh2 can you hit the button so i don't break things? =D

**Review by raymondh2 — APPROVED** [review]

**Pull request merged**